test/Microsoft.AspNetCore.Blazor.Test/Routing/RouteTableTests.cs

```diff
@@ -305,10 +305,21 @@ public TestRouteTableBuilder AddRoute(string template, Type handler = null)
                 return this;
             }
 
-            public RouteTable Build() => new RouteTable(_routeTemplates
-                .Select(rt => new RouteEntry(TemplateParser.ParseTemplate(rt.Item1), rt.Item2))
-                .OrderBy(id => id, RouteTable.RoutePrecedence)
-                .ToArray());
+            public RouteTable Build()
+            {
+                try
+                {
+                    return new RouteTable(_routeTemplates
+                        .Select(rt => new RouteEntry(TemplateParser.ParseTemplate(rt.Item1), rt.Item2))
+                        .OrderBy(id => id, RouteTable.RoutePrecedence)
+                        .ToArray());
+                }
+                catch (InvalidOperationException ex) when (ex.InnerException is InvalidOperationException)
+                {
+                    // ToArray() will wrap our exception in its own.
+                    throw ex.InnerException;
```

Comment:
The behavior changed here and I'm not 100% certain why - it's possible that KoreBuild is now using netcoreapp2.1 to run tests.

Reply:
That's strange. Probably not something to worry about though, as it's just changing the structure of the exceptions in the case of a fatal error. Nobody could be relying on the old behavior.